andela/trime
# Feature/empty orgs

#### What does this PR do?

Adds a message to the orgs page to show when the org has no projects.

#### How should this be manually tested?

Click on `CREATE ORG` in the Trime dashboard, after providing a title and description click on `SUBMIT`. The new organisation will be displayed on the dashboard. Clicking on the organisation's name should display the page below.

#### What are the relevant pivotal tracker stories?

[Feature #119630225] (https://www.pivotaltracker.com/projects/1454736/stories/119630225)

<img width="1680" alt="screen shot 2016-06-29 at 11 06 42 am" src="https://cloud.githubusercontent.com/assets/17295379/16445209/3d26a794-3dea-11e6-893e-bda49d8202b0.png">


@andela-ladenusi
